docs/get-started/first-api-call.md

```diff
@@ -17,24 +19,29 @@ To start using the APIs you need to tell the system that you are allowed to do s
 !!! danger
     Remember that the API Key and API Secret are for personal use only. Do not share them with anyone.
 
-All you need to do is to attach an alphanumeric string called **token** to your calls. This token will identify you as valid user and is generated from two parameters: `API Key` and `API Secret`. In order to get them, and once your account has been verified, you can [sign in](https://developers.amadeus.com/signin) and follow these steps:
+All you need to do is to attach an alphanumeric string called **token** to your calls. This token will identify you as a valid user and is generated from two parameters: `API Key` and `API Secret`. To get them, and once your account has been verified, you can [sign in](https://developers.amadeus.com/signin) and follow these steps:
+
+* Click on your username \(top right corner\) 
+* Go to [My Self-Service Workspace](https://developers.amadeus.com/my-apps) 
+    ![api_key1](../images/api_key1.png)
```

Comment:
this pic looks a bit blurry

Reply:
can't resolve it yet ... to be investigated further once we have more pics